docs/source/getting-started/examples/accuracy.ipynb

```diff
@@ -0,0 +1,269 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Accuracy Method"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Many prior examples of queries used comparisons of the actual values to the computed differentially private values. Intuitively, we assessed the quality of our results by visually inspecting the proximity of the actual values to the differentially private outputs. \n",
+    "\n",
+    "However, this approach isn't concrete since interpretations can vary widely on scale, type of visualization, and other factors. A more objective way to evaluate differentially private outputs is by using the `accuracy` method. \n",
```

Comment:
I'd emphasize that comparing one DP-release to the true value is a comparison with exactly one draw from a random variable: You might get "lucky" and get a release that is much more accurate than usual, or "unlucky" and get a release that is much less accurate than usual.

Also: with synthetic data a comparison with the "true" value is possible... but in the real world you can't do that and maintain privacy. Even if you don't release these intermediate results, if your analysis plan is influenced by what you see in the real data, that's a privacy leak.